rfp-proposals/ars-js-filecoin-explorer.md

```diff
@@ -0,0 +1,350 @@
+| --- | --- | --- | --- |
+| 1 | Finish front-end develop. | [@maxi-zhang](https://github.com/maxi-zhang) | $3,750 |
+| 2 | Finish Back-end develop. |  [@waynewyang](https://github.com/waynewyang) | $3,750 |
+
+
+
+**M3（2019.12.23-2020.01.10）:**
+
+- **Total Funding Amount:$5,000**
+
+| No. | Software Functionality | Team Member | Funding Required |
+| --- | --- | --- | --- |
+| 1    | All project deliverables are completed and added to the [Filecoin Shipyard](https://github.com/filecoin-shipyard) (website, documentation, codebase),Pre-delivery testing | [@maxi-zhang](https://github.com/maxi-zhang)<br /> [@waynewyang](https://github.com/waynewyang) | $2,500 |
+
+
+
+**M4（2020.01.12-2020.12.31）:**
+
+- **Total Funding Amount:$10,000**
+
+| No.  | Software Functionality  | Team Member      | Funding Required |
+| ---- | ----------------------- | ----------- | ----------------- |
+| 1    | Maintenance and Upgrade | Arsyun team | $5,000          |
+
+
+
+## Total Budget Requested
+
+| Milestones | Funding Amount |
+| --- | --- |
+| M1 | $10,000 |
+| M2 | $7,500 |
+| M3 | $2,500 |
+| M4 | $5,000 |
+| **Total** | **$25,000** |
+
+
+
+## Maintenance and Upgrade Plans
+
+- Interesting on long term maintenance and upgrade plan
+
+  
+
+# Team
+
+## Team Members
+- [@fengyuxilu](https://github.com/fengyuxilu): pm/ue
+- [@Frank-design](https://github.com/Frank-design): ui
+- [@maxi-zhang](https://github.com/maxi-zhang) : front-end
+- [@waynewyang](https://github.com/waynewyang) : arch/back-end
+
+## Team Member LinkedIn Profiles
+
+- [waynewyang](https://www.linkedin.com/in/waynewyang/?locale=en_US)
+
+
+## Team Website
+
+- https://www.arsyun.com
+- https://github.com/arsyun
+
+## Relevant Experience
+
+- ARS team has rich experience in distributed cloud storage and has deployed practical products. And provides cloud service base on IPFS and filecoin.
+  - [Essential-Thinking-of-Blockchain](https://github.com/arsyun/Essential-Thinking-of-Blockchain)
+  - [Product:pServer](https://www.arsyun.com/web/index.html)
+-  [@waynewyang](https://github.com/waynewyang) is the [Contributor](https://github.com/filecoin-project/go-filecoin/graphs/contributors) of go-filecoin, we can deeply understand the work flow about file coin.
+- ARS Team has been attended IPFS camp in berlin and Barcelona, We're very interesting in IPFS and filecoin, and plan to do some application based on IPFS & filecoin.
+
+## Team code repositories
+
+- https://github.com/arsyun.
+- https://github.com/arsyun/js-filecoin-explorer
+
+# Additional Information
+
+**Now, We've alreay do some pre work**,
+
+- [Demo code of ars-js-filecoin-explorer](https://github.com/arsyun/js-filecoin-explorer)
+  - Front-end: React
+  - Back-end: Nodejs
+- [ Backend db design of ars-js-filecoin-explorer ](https://github.com/arsyun/js-filecoin-explorer/tree/master/db)
+  - Mongodb, Read > Write.
+- [Ue draft of ars-js-filecoin-explorer ]( https://wbhnrp.axshare.com )
```

Comment:
Great to see that some pre-work has already been done. However, the mockups look like they focus more on network activity than producing a chain explorer.

Reply:
It's just a draft UE, actually our explorer including chian explorer and network activity. At present, the chain explorer  is in Network Overview,we'll  perfect it later.